lib/modules/yaml_parser.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require 'yaml'
+
+class YamlParser
+  class << self
+    def configuration
+      cops_name_array
+    rescue Errno::ENOENT
+      nil
```

Comment:
Maybe it would be better to show some error to user in this case, WDYT?

Reply:
I need to get nil, because if no file with configuration It check files by default settings